packages/packages/recently-edited/src/components/top-bar/posts-list.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+	MenuItem,
+	ListSubheader,
+	Typography,
+} from '@elementor/ui';
+
+import { __, isRTL } from '@wordpress/i18n';
+import DocTypeChip, { Props } from '../top-bar/chip-doc-type';
+import { Post } from '../../hooks/use-recent-posts';
+
+export type RecentPostsProps = {
+	recentPosts: Post[];
+};
+
+export default function PostsList( { recentPosts }: RecentPostsProps ) {
+	return (
+		<>
+			<ListSubheader sx={ { fontSize: 12, fontStyle: 'italic', fontWeight: 300, pl: 4 } } component="div" id="nested-list-subheader">
+				{ __( 'Recent', 'elementor' ) }
+			</ListSubheader>
+
+			{ recentPosts.length
+				? ( recentPosts.map( ( { title, edit_url: editUrl, type, id } ) => (
+					<MenuItem
+						key={ id }
+						component="a"
+						href={ editUrl }
+					>
+						{ title }
+						<DocTypeChip postType={ type.post_type } docType={ type.doc_type as Props['docType'] } label={ type.label } />
+					</MenuItem>
+				) ) ) : (
+					<Typography variant="caption" sx={ { color: 'grey.500', fontStyle: 'italic', fontWeight: 300, p: 4, ...( isRTL() ? { pl: '25px' } : { pr: '25px' } ) } } component="div" aria-label={ undefined }>
```

Comment:
1. No need to handle the RTL flip, Material does this for you, just use the LTR version value.
2. Please don't use hard-coded values (25px), use the spacing values from the Storybook (the value of 7 will be 24px for example).
3. Are you sure you need the font-weight 300?